### PR TITLE
[HW_9][JS_7]

### PR DESCRIPTION
**Pull request by JackBlaaack**
Work with Classes

### CODE REVIEW

**Review by anatoly-karpovich — DISMISSED** [review]

**Review by VladimirLQA — DISMISSED** [review]

File: JS_7/task2.8.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  addAnimal(animal) {
+    try {
+      if (!(animal instanceof Animal)) {
+        throw new Error("This is not an animal");
+      }
+      if (animal instanceof Snake) {
+        throw new Error(`There will be no snakes, mister Potter!`);
+      }
+      this.animals.push(animal);
+      return `Success! Add ${JSON.stringify(animal)}`;
```
Comment (anatoly-karpovich) [inline JS_7/task2.8.js:38]: So not return this-like strings. If function is void - no return is needed

File: JS_7/task2.8.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  }
+
+  removeAnimal(id) {
+    try {
+      const index = this.animals.findIndex((animal) => animal.id === id);
+      if (index === -1) {
+        throw new Error("Animal not found");
+      } else {
+        this.animals.splice(index, 1);
+        return `Animal with id ${id} removed successfully`;
```
Comment (anatoly-karpovich) [inline JS_7/task2.8.js:65]: Same

File: JS_7/task2.7.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      try {
+        if (!(animal instanceof Animal)) {
+          throw new Error('error');
+        }
+        if (animal instanceof Snake) {
+            throw new Error(`There will be no snakes, mister Potter!`)
+        }
+        if (this.animals?.length === 0) this.animals = [];
+        this.animals.push(animal);
+        return `Success! Add ${JSON.stringify(animal)}`
```
Comment (anatoly-karpovich) [inline JS_7/task2.7.js:32]: Same

**Review by anatoly-karpovich — DISMISSED** [review]

**Review by VladimirLQA — DISMISSED** [review]

File: JS_7/task2.8.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+// 8. Добавьте методы removeWorker() и removeAnimal() // Подумайте, как будем удалять, по какому полю будем выбирать:)
+
+class Zoo {
```
[the remark this answers is not in the document]
Comment (JackBlaaack) [reply JS_7/task2.8.js:3]: Explain me, what do you mean? each file has own class Zoo, I initialized the project.

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**